packages/rocketchat-oembed/assets/stylesheets/oembedAdmin.less

```diff
@@ -0,0 +1,94 @@
+.oembed-view {
+	z-index: 15;
```

Comment:
Remove these 3 lines completely as it's causing a double scroll bar:

![image](https://user-images.githubusercontent.com/187449/32149594-11a1fa46-bd5b-11e7-876a-641f3e16af2b.png)

```
.oembed-view {
    z-index: 15;
    overflow-y: auto;
    overflow-x: hidden;
}
```